test/config_generator_test.rb

```diff
@@ -0,0 +1,54 @@
+require "test_helper"
+
+class ConfigGeneratorTest < Minitest::Test
+  include TestHelper
+
+  def test_generate
+    assert_yaml "test_generate.yml",
+                tools: [],
+                comment_out_lines: [8..12, 15..19]
+  end
+
+  def test_generate_with_tools
+    # NOTE: Use all the tools to check schema for the whole example.
+    tools = Runners::Processor.children.filter_map do |id, klass|
+      begin
+        klass.config_example
+        id
+      rescue NotImplementedError
+        nil
+      end
+    end
+
+    assert_yaml "test_generate_with_tools.yml",
+                tools: tools,
+                comment_out_lines: [9..403, 406..410, 413..417]
+  end
+
+  private
+
+  def assert_yaml(expected_filename, tools:, comment_out_lines:)
+    actual = Runners::ConfigGenerator.new.generate(tools: tools)
+
+    assert_equal data(expected_filename).read, actual
+
+    # Comment out
```

Comment:
Maybe, this action is uncommenting.

```suggestion
    # Uncomment
```